weather_mv/loader_pipeline/bq.py

```diff
@@ -94,6 +95,10 @@ class ToBigQuery(ToDataSink):
     skip_region_validation: bool
     disable_grib_schema_normalization: bool
     coordinate_chunk_size: int = 10_000
+    create_polygon: bool = False
+    should_create_polygon: bool = False
```

Comment:
Can this be one flag?

Reply:
Could you please review the discussion at https://github.com/google/weather-tools/pull/337#discussion_r1246421324 and provide your feedback? I will make the necessary changes to the flag based on your input.

Reply:
Code updated as per your suggestion. Thanks for the input!